templates/ldap.conf.j2

```diff
@@ -0,0 +1,34 @@
+# {{ ansible_managed }}
```

Comment:
I don't see any differences to `ldap-master.conf.j2`. Remove one file if this is the case

Reply:
It's not. The filter variable is different iirc